CFH robot example ros

shehzi001 opened the pull request.

shehzi001 committed:
refactor meta information and add message generation dependencies

shehzi001 committed:
add configurations to compile  robot example using catkin tool

shehzi001 committed:
add ros example for cfh robot

shehzi001 committed:
add attentation and camera control ROS messages

moriarty commented:
Looks good but do you want it merged now? 

Or will you complete the example. It would be good to have all the features in this one:
at_work_central_factory_hub/rockin/tools/rockin-fake-robot.cpp 
on the rockin2015 branch 

https://github.com/rockin-robot-challenge/at_work_central_factory_hub/blob/rockin2015/rockin/tools/rockin-fake-robot.cpp

shehzi001 committed:
[robot_example_ros] remove unsupported compressed image message headers

shehzi001 committed:
[robot_ros_example] replace data type of the fields of  attention message  with ROS std_msgs data types

shehzi001 committed:
[robot_example_ros] remove quality control camera subscriber

shehzi001 committed:
[robot_example_ros] register benchmarkstate, drillmachine and conveyor belt messages with refbox

shehzi001 committed:
[robot_example_ros] add ros messages related to benchmark, drillmachine and conveyor belt etc.

shehzi001 committed:
[robot_example_ros] remove camera control message

shehzi001 committed:
[robot_example_ros] modify definition of Benchmarkstate message and add BenchmarkScenerio message

shehzi001 committed:
[robot_example_ros] advertise ROS topics
    To publish benchmark state, drill machine and conveyor belt status, inventory and order messages from refbox.

shehzi001 committed:
[robot_example_ros] add message BenchmarkScenerio to message generation list

shehzi001 committed:
[robot_example_ros] add subscribers to send command to conveyor belt and drill machine, and feedback messages to refbox

sthoduka committed:
fix publisher names

sthoduka committed:
define integers for benchmark states in ros msg

shehzi001 committed:
Merge pull request #1 from sthoduka/cfh_robot_example_ros

Cfh robot example ros
